libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -742,12 +742,18 @@ bool ASTJsonExporter::visit(WhileStatement const& _node)
 
 bool ASTJsonExporter::visit(ForStatement const& _node)
 {
-	setJsonNode(_node, "ForStatement", {
+
+	std::vector<std::pair<std::string, Json::Value>> attributes = {
 		std::make_pair("initializationExpression", toJsonOrNull(_node.initializationExpression())),
 		std::make_pair("condition", toJsonOrNull(_node.condition())),
 		std::make_pair("loopExpression", toJsonOrNull(_node.loopExpression())),
 		std::make_pair("body", toJson(_node.body()))
-	});
+	};
+
+	if (_node.annotation().isSimpleCounterLoop.set())
+		attributes.emplace_back(std::make_pair("isSimpleCounterLoop", *_node.annotation().isSimpleCounterLoop));
```

Comment:
Does this need a `std::make_pair`? Wouldn't the following work just as fine (in fact, using `make_pair` here just turns the `emplace_back` into a `push_back`).

```suggestion
		attributes.emplace_back("isSimpleCounterLoop", *_node.annotation().isSimpleCounterLoop);
```

Reply:
Indeed, `std::make_pair` is not necessary here and not coherent with `emplace_back`.
Fixed.